client/shipping-label/components/label-purchase-modal/step-confirmation-button/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormButton from 'components/forms/form-button';
+
+const SetpConfirmationButton = ( { disabled, onClick, children } ) => {
+	return (
+		<div className="step-confirmation-button">
+			<FormButton
+				type="button"
+				onClick={ onClick }
+				disabled={ Boolean( disabled ) }
+				isPrimary >
+				{ children }
+			</FormButton>
+		</div>
+	);
+};
+
+SetpConfirmationButton.propTypes = {
+	disabled: PropTypes.bool,
+	onClick: PropTypes.func.isRequired,
+};
+
+export default SetpConfirmationButton;
```

Comment:
LOL good catch, fixed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/ee4d40e101ccf3030d14002090f26904d4e42288